server/controllers/profile.controller.js

```diff
@@ -41,9 +47,11 @@ class ProfileController {
       username,
       bio,
       image,
-      articles: `${userActivity[0]} articles written.`,
-      // following: `You are following ${userActivity[1]} authors.`,
-      // followers: `You have ${userActivity[2]} followers.`
+      articlesWritten: `${userActivity[0].count} articles written.`,
```

Comment:
Nice work @TheDrizzyWay. Should we return just the figures instead? So the frontend has flexibility in the presentation.